### ISSUE TITLE
Create Zet shortcode

### ISSUE THREAD

**Issue by danielmichaels** [danielmichaels/danielms#53]
My Zet's are a link directly to their location in GitHub.

To keep people on the page and reduce the load times it should be possible to do either,

- grab the raw text from github and generate a single page with it
- pull out text data from zets.json and load that via a shortcode

Either of these would keep zet data on the site and significantly reduce the load times. Plus its jarring linking directly to github due to the page transition.

**Comment by danielmichaels** [comment]
Sample of `zet.json`

```json
[
	{
		"name": "README.md",
		"path": "20220424000235",
		"sha": "ecff2486607fd1347aa6dd92568f716755172750",
		"size": 618,
		"url": "https://api.github.com/repos/danielmichaels/zet/contents/20220424000235/README.md?ref=main",
		"html_url": "https://github.com/danielmichaels/zet/blob/main/20220424000235/README.md",
		"git_url": "https://api.github.com/repos/danielmichaels/zet/git/blobs/ecff2486607fd1347aa6dd92568f716755172750",
		"download_url": "https://raw.githubusercontent.com/danielmichaels/zet/main/20220424000235/README.md",
		"type": "file",
		"content": "How to check Go module versions",
		"encoding": "base64",
		"date": "2022-04-24T00:00:00Z",
		"_links": {
			"self": "https://api.github.com/repos/danielmichaels/zet/contents/20220424000235/README.md?ref=main",
			"git": "https://api.github.com/repos/danielmichaels/zet/git/blobs/ecff2486607fd1347aa6dd92568f716755172750",
			"html": "https://github.com/danielmichaels/zet/blob/main/20220424000235/README.md"
		}
	},
]
````

Using the `download_url` it is possible to grab all the text. The template data can be populated using `zet` info from the `zet.json` file.

**Comment by danielmichaels** [comment]
Instead of a shortcode just use the `single.html` via hugo.

closed by #54 and #55
